server/routes/addPatient/addPatientCompleted.ts

```diff
@@ -0,0 +1,32 @@
+import { Request, Response } from 'express'
+import PrisonerSearchService from '../../services/prisonerSearchService'
+import HospitalSearchService from '../../services/hospitalSearchService'
+
+export default class AddPatientCompletedRoutes {
+  constructor(
+    private readonly prisonerSearchService: PrisonerSearchService,
+    private readonly hospitalSearchService: HospitalSearchService
+  ) {}
+
+  private renderView = async (req: Request, res: Response): Promise<void> => {
+    const prisonerNumber = req.query.prisonerNumber as string
+    const hospitalId = req.query.hospitalId as string
+    const { user } = res.locals
+
+    const [hospital, prisoner] = await Promise.all([
+      this.hospitalSearchService.getHospital(hospitalId, user),
+      this.prisonerSearchService.getPrisonerDetails(prisonerNumber, user),
```

Comment:
I was just wondering if there is any scenario where a user is only allowed to add prisoners from their current caseload? If so should we be searching by caseload here?

Reply:
prisoners are `OUT` though